features/scroll-markers.yml

```
@@ -0,0 +1,11 @@
+name: Scroll markers
```

```suggestion
name: `::scroll-marker` and `::scroll-marker-group`
```

would be more consistent with other pseudo-element features.

---

Ah, but come to think of it, this feature also contains the `scroll-marker-group` **property**, so the feature isn't just about pseudos.
I think "Scroll markers" (your initial proposal) is therefore better.